internal/olm/operator/internal/deployment.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
+	"fmt"
+
+	registryutil "github.com/operator-framework/operator-sdk/internal/util/operator-registry"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// The image operator-registry's initializer and registry-server binaries
+	// are run from.
+	// QUESTION(estroz): version registry image?
+	registryBaseImage = "quay.io/openshift/origin-operator-registry:latest"
```

Comment:
It [looks like](https://quay.io/repository/openshift/origin-operator-registry?tag=latest&tab=tags) the tags are currently based on openshift versions, which is kind of a bummer, since it is possible to run OLM and this subcommand on non-OpenShift clusters.

Ideally we could lookup the version of OLM currently installed and then know which operator-registry image version to use.

In the absence of a better solution, `:latest` seems reasonable for now (especially while under an `alpha` subcommand)